SU-SWS/ace-casbsgryphon
# CAS-187: jump link attachments for people views

# READY FOR REVIEW

# Summary
- This PR has Bruce's Current Fellows and Staff views with Jump Links (like old site!)

# Review By (Date)
- Soon!

# Urgency
-  Let's get it merged!

# Steps to Test

1. Throw a list on a page
2. Pick the current fellows or Staff view
3. Filter the current fellows by year, if desired
4. See a jump list
5. Note: the staff view doesn't current produce anything

# Affected Projects or Products
- No

# Associated Issues and/or People
- @brucedyoung 
- CAS-187

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
